modules/external-api/pom.xml

```diff
@@ -261,7 +261,8 @@
               OSGI-INF/external-statistics-endpoint.xml,
               OSGI-INF/external-workflow-definitions-endpoint.xml,
               OSGI-INF/external-workflows-endpoint.xml
-            </Service-Component>
+            <Service-Component-->
+            <_dsannotations-options>norequirements</_dsannotations-options>
```

Comment:
In that case, the CatalogUIAdapterFactory (ManagedServiceFactory) registers java classes as OSGi services. 
CommonEventCatalogUIAdapter, ...

Reply:
Ugh, that's unfortunate. I would include a comment with the reasoning here then, since I can see someone coming along and stripping this out otherwise.